### PR TITLE
Added BGP Remote peer link and graph in Routing Overview

### PR DESCRIPTION
**Pull request by tim427**
Added BGP Remote peer link and graph in Routing Overview;

Screenshot 2023-10-30 at 09 04 35
Screenshot 2023-10-30 at 09 08 19

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by tim427** [comment]
`dbFetchRow` is used, like the rest of this file. Refactoring to newer Laraval Eloquent will be a project for during holidays in December ;)

File: includes/html/pages/routing/bgp.inc.php [hunk 1]
```diff
@@ -313,11 +313,12 @@
         }
 
         unset($sep);
+        $peer_remote = dbFetchRow('SELECT D.device_id FROM `bgpPeers` AS `B`, `devices` AS `D` WHERE `B`.`device_id` = `D`.`device_id` AND B.bgpLocalAddr = ?', [$peer['bgpPeerIdentifier']]);
```
Comment (murrant) [inline includes/html/pages/routing/bgp.inc.php:316]:
```suggestion
        $peer_remote = BgpPeer::where('bgpLocalAddr', $peer['bgpPeerIdentifier'])->value('device_id');
```

Comment (tim427) [reply includes/html/pages/routing/bgp.inc.php]: Super! I've changed the code, it works (of course ;) ) and updated the PR!

(Nice to see the (for me) new Eloquent-way in action -> let's rebase this section later on)

**Comment by tim427** [comment]
This PR consists commits from others, new attempt in #15532

**Pull request closed without merging**